spring-thymeleaf-2/src/main/resources/templates/mvcdata/email-bean-data.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+	  xmlns:th="http://www.thymeleaf.org">
+<h1>Subject</h1>
+<p th:text="${@emailData.emailSubject}">Subject</p>
+</br>
```

Comment:
I believe this would be better as `<br/>` since that would make it well-formed.

Reply:
I removed the breakers